.pre-commit-config.yaml

```diff
@@ -14,6 +14,7 @@ repos:
       - id: check-json
         exclude: ^\.devcontainer/|^src/.*/tests/resources/.*$ # Exclude .devcontainer folder to keep the comments in the json files, exclude test resources
       - id: check-added-large-files
+        exclude: src/featuremap/tests/resources/ppmSeq_chr1_alt2_8.test_size_0.1.xgb_model.json
```

Comment:
add also the standard model

Reply:
what is the file size? we don't want to store large files in the repo
need to consider make a smaller size file for test if possible
or store the file in git lfs